ncl/ncl_index/ncl-index-table.csv

```diff
@@ -11,3 +11,15 @@ NCL Function,Description,Python Equivalent,Notes
 `day_of_week <https://www.ncl.ucar.edu/Document/Functions/Built-in/day_of_week.shtml>`__,"Calculates the day of the week given month, day, and year","``cftime.datetime().dayofwk``",`details <../ncl_entries/day_of_week.ipynb>`__
+`abs <https://www.ncl.ucar.edu/Document/Functions/Built-in/abs.shtml>`__,"Returns the absolute value of numeric data","``abs()`` or ``numpy.abs()``",`Example Notebook <../ncl_entries/general_applied_math.ipynb#abs>`__
+`avg <https://www.ncl.ucar.edu/Document/Functions/Built-in/avg.shtml>`__,"Computes the average of a variable regardless of dimensionality","``numpy.average()`` or ``numpy.mean()``",`Example Notebook <../ncl_entries/general_applied_math.ipynb#avg>`__
+`cumsum <https://www.ncl.ucar.edu/Document/Functions/Built-in/cumsum.shtml>`__,"Calculates the cumulative sum","``numpy.cumsum()``",`Example Notebook <../ncl_entries/general_applied_math.ipynb#cumsum>`__
+`decimalPlaces <https://www.ncl.ucar.edu/Document/Functions/Contributed/decimalPlaces.shtml>`__,"Truncates or rounds to the number of decimal places specified","``round()`` or ``numpy.round()``",`Example Notebook <../ncl_entries/general_applied_math.ipynb#decimalPlaces>`__
+`get_d2r <https://www.ncl.ucar.edu/Document/Functions/Contributed/get_d2r.shtml>`__,"Return a constant that converts degrees to radians","``math.pi / 180``",`Example Notebook <../ncl_entries/general_applied_math.ipynb#get-d2r>`__
+`get_r2d <https://www.ncl.ucar.edu/Document/Functions/Contributed/get_r2d.shtml>`__,"Return a constant that converts radians to degrees","``180 / math.pi``",`Example Notebook <../ncl_entries/general_applied_math.ipynb#get-r2d>`__
+`get_pi <https://www.ncl.ucar.edu/Document/Functions/Contributed/get_pi.shtml>`__,"Return pi as a type float or double","``math.pi``",`Example Notebook <../ncl_entries/general_applied_math.ipynb#get-pi>`__
+`log <https://www.ncl.ucar.edu/Document/Functions/Built-in/log.shtml>`__,"Computes the natural log of a numeric type","``math.log()``",`Example Notebook <../ncl_entries/general_applied_math.ipynb#log>`__
+`log10 <https://www.ncl.ucar.edu/Document/Functions/Built-in/log10.shtml>`__,"Computes the log base 10 of a numeric type","``math.log10()``",`Example Notebook <../ncl_entries/general_applied_math.ipynb#log10>`__
+`mod <https://www.ncl.ucar.edu/Document/Functions/Contributed/mod.shtml>`__,"Remainder function which emulates the Fortran 'mod' intrinsic function","``%`` or ``numpy.mod()``",`Example Notebook <../ncl_entries/general_applied_math.ipynb#mod>`__
+`product <https://www.ncl.ucar.edu/Document/Functions/Built-in/product.shtml>`__,"Computes the product of the input","``math.prod()`` or ``numpy.prod()``",`Example Notebook <../ncl_entries/general_applied_math.ipynb#product>`__
+`sum <https://www.ncl.ucar.edu/Document/Functions/Built-in/sum.shtml>`__,"Sums the input","``sum()`` or ``numpy.sum()``",`Example Notebook <../ncl_entries/general_applied_math.ipynb#sum>`__
```

Comment:
Super minor thing, but it looks like the text in the notes section should probably be lowercase for consistency.

Reply:
Sure thing!